src/main/src/main/java/org/geoserver/logging/Log4jUtils.java

```diff
@@ -0,0 +1,705 @@
+/* (c) 2014 - 2016 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
```

Comment:
Where is this file from to have such an old header?

Reply:
I removed the older copyright from the header.